akka-docs/versions.json

```diff
@@ -0,0 +1,66 @@
+{
+  "2.1.": {
+    "latest": "2.1.4",
+    "outdated": true,
+    "instead": "2.4."
+  },
+  "2.2.": {
+    "latest": "2.2.5",
+    "outdated": true,
+    "instead": "2.4."
+  },
+  "2.3.": {
+    "latest": "2.3.13",
+        "outdated": true,
+        "instead": "2.4."
+  },
+  "2.4.": {
+    "latest": "2.4.0"
+  },
```

Comment:
this means that browsing 2.4.0 docs will also show the warning, which it should not